pkg/policy/l4.go

```diff
@@ -185,6 +185,8 @@ type L4Filter struct {
 	Protocol api.L4Proto `json:"protocol"`
 	// U8Proto is the Protocol in numeric format, or 0 for NONE
 	U8Proto u8proto.U8proto `json:"-"`
+	// wildcard is the cached selector representing a wildcard in this filter, if any.
```

Comment:
Might be helpful for future developers to note what this means in terms of higher-layer policies, which cases this wildcard field would be set, etc.

Reply:
```
	// wildcard is the cached selector representing a wildcard in this filter, if any,
	// This is nil when none of the selectors in 'L7RulesPerSelector' is a wildcard.
	// When there is a wildcard selector in 'L7RulesPerSelector' this is set to that
	// same selector, which can then be used as a map key to find the correspondng L7
	// policy.
```